App.tsx

```diff
@@ -474,6 +475,16 @@ const BaseApp: React.FunctionComponent<{
     );
   }
 
+  const flip = (data: Record<AvalancheCenterID, string>) => Object.fromEntries(Object.entries(data).map(([key, value]) => [value, key]));
+
+  // get the universal link the app was open with, if one exists
+  let initialUrl: string | null = null;
+  Linking.getInitialURL()
+    .then(url => {
+      initialUrl = url;
+    })
+    .catch(() => logger.error('Getting initialUrl failed'));
+
   const linking = {
     prefixes: [AvalancheCenterWebsites['NWAC'] + '/observations/#/view/'],
     config: {
```

Comment:
Adding this under `Observations` might fix your issue with not having the list:

https://reactnavigation.org/docs/configuring-links/#rendering-an-initial-route

Reply:
you are right, if I add this --- then the Observation List screen is there if a universal link is open before app has been open/Obs page has been open. The back button will take you back to the list, not the home screen. If that is what we want - we can add that, lets check with Charlotte tomorrow

Reply:
Charlotte to play with this and let us know if it should change or not